src/errors/handling.ts

```diff
@@ -9,7 +9,8 @@ export const handleApiError = async (
   customMessage?: (
     statusCode: number,
     rawMessageText: string
-  ) => Promise<string>
+  ) => Promise<string>,
+  url?: string
```

Comment:
This is the fallback `url` which is used if the `ResponseError` doesn't have it, for whatever reason.

I spent a lot of time trying to track down why this was happening and it felt like something in the generated runtime. The only test suite that was failing was `errorHandling.test.ts`. When running edge vs. node locally and using the debug environment values, the responses were identical.